MAUI/DataGrid/Column-types.md

```diff
@@ -1418,6 +1424,168 @@ The `DataGridNumericColumn` allows formatting the numeric data with culture-spec
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name = "dataGrid"
+                       ItemsSource = "{Binding Orders}"
+                       AllowEditing = "True"
+                       SelectionMode = "Single"
+                       AutoGenerateColumnsMode = "None"
+                       NavigationMode="Row">
+    <syncfusion:SfDataGrid.Columns>
+        <syncfusion:DataGridMultiColumnComboBoxColumn AutoGenerateColumnsMode = "None"
+                                                      MappingName = "CustomerID"
+                                                      ItemsSource = "{Binding OrderDetails}"
+                                                      DisplayMember = "CustomerID"
+                                                      ValueMember = "CustomerID" 
+                                                      HeaderText="Customer ID">
+            <syncfusion:DataGridMultiColumnComboBoxColumn.Columns>
+                <syncfusion:DataGridTextColumn MappingName = "CustomerID" HeaderText = "Customer ID"/>
+                <syncfusion:DataGridNumericColumn MappingName = "ProductID" HeaderText = "Product ID"/>
+            </syncfusion:DataGridMultiColumnComboBoxColumn.Columns>
+        </syncfusion:DataGridMultiColumnComboBoxColumn>
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+
+{% endhighlight %}
+
+{% highlight c# %}
+var column = new DataGridMultiColumnComboBoxColumn()
+{
+    MappingName = "CustomerID",
+    HeaderText = "Customer ID",
+    ValueMember = "CustomerID",
+    DisplayMember = "CustomerID",
+    ItemsSource = viewModel.OrderDetails,
+    AutoGenerateColumns = false,
+    Columns = new ColumnCollection()
+    {
+        new DataGridTextColumn(){MappingName = "CustomerID"},
+        new DataGridNumericColumn(){MappingName = "ProductID"}
+    }
+};
+this.dataGrid.Columns.Add(column);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+SfDataGrid triggers, (CurrentCellDropDownSelectionChanged)[] event, when the SelectedValue is changed. (CurrentCellDropDownSelectionChangedEventArgs)[] of `CurrentCellDropDownSelectionChanged` event provides the information about the changed cell value.
+
+`SelectedIndex` property returns the index of selected item.
+`SelectedItem` property returns the selected item from drop down list.
+
+<img alt="DataGridMultiColumnComboBox column .NET MAUI DataGrid" src="Images\column-types\maui-datagrid-MultiColumn-ComboBox-column.png" width="404" height="396"/>
+
+### Auto-complete support
+
+You can allow SfMultiColumnComboBox to complete the entered input value automatically by setting the (AllowAutoComplete)[] property to `true`. `AllowAutoComplete` property will autocomplete the input value by comparing it to each item of the underlying data source of `DataGridMultiColumnComboBoxColumn` and returns the matched value by DisplayMember.
```

Comment:
You can enable the SfMultiColumnComboBox to automatically complete the entered input by setting the AllowAutoComplete property to true. When enabled, this property compares the entered text with each item in the underlying data source of DataGridMultiColumnComboBoxColumn and autocompletes the input with the matched value based on the DisplayMember.